mailpile/spambayes/OptionsClass.py

```diff
@@ -89,6 +89,8 @@
     except ImportError:
         import io as StringIO
 
+    xrange = range
```

Comment:
Not sure how many similar cases like this there are. But an idea is to have a `py3.py` (or other naming) file and from that file you import all the the methods that have changed. 

That way it'll all be collected in one single file rather than a bit spread out.

Reply:
The proper thing to do would probably be to use [six](https://pythonhosted.org/six/) which is listed in requirements-with-deps.txt. Seems like there is a desire to avoid external dependencies as much as possible, so I don't know if it would be acceptable to turn that into a direct dependency or not. I'd be for it.

It appears it's only use in this case would produce a 128 integer array, and it would only happen on import. Using a generator there doesn't seem like a huge win. I could just convert it to use `range()`. My goal in all cases was to try to stay as close to the original as possible. I don't know how much that matters.

Reply:
There is a desire to avoid external dependencies that complicate packaging work. I don't think six is a problem, it should be everywhere by now.